bundle-workflow/src/test_workflow/test_results.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from aws.s3_bucket import S3Bucket
+
+
+class TestResults:
+    def __init__(self, bundle_manifest, test_recorder):
+        self.bundle_manifest = bundle_manifest
+        self.test_recorder = test_recorder
+
+    def to_s3(self, bucket):
+        """
+            Publishes tests results to S3 pulling information from {self.test_recorder}
+            And cleans up all local storage after publishing ({self.test_recorder}.clean_up())
+        """
+        s3_bucket = S3Bucket(bucket, '<role-arn>', 'test-publisher-session')
```

Comment:
I think we already have a role defined. @gaiksaya could help and chime in with the role we have already created in Jenkins?

Reply:
Yes. Asked @owaiskazi19 to remove this params except the s3bucket. They are not required.